pypeit/scripts/sensfunc.py

```diff
@@ -65,7 +66,8 @@ def get_parser(cls, width=None):
                                  "F|    [sensfunc]\n"
                                  "F|         flatfile = Calibrations/Flat_A_0_DET01.fits\n"
                                  "\nWhere Flat_A_0_DET01.fits is the flat file in your Calibrations directory\n")
-
+        parser.add_argument("--telgridfile", type=str, help=argparse.SUPPRESS)
+        parser.add_argument("--polyorder", type=int, help=argparse.SUPPRESS)
```

Comment:
I actually wouldn't suppress these.  If you found it helpful, others likely will as well.

Reply:
Since these options are more easily settable with the `--sensfile` option when called from the command line, I chose to not muddy the `--help` interface with these.  The purpose of adding them at all is for calling the `SensFunc` class from another python script.

If you still think I should include them in the user-facing interface, I will add proper help strings.

Reply:
Is there a reason you're calling this full script from a different python script, instead of calling the `sensfunc.SensFunc` class directly?

Reply:
This script sets up the FITS header for the `sensfunc` file and nicely wraps the functionality of the `sensfunc.SensFunc` class.  I was calling the script from a script to automate making many many sensitivity function files for comparison purposes (conditions, stars, airmasses, etc.).  I'm not sure how often users will be doing this kind of thing.  Maybe I just remove these and work through a `--sensfile` like the command-line version.

Reply:
Would your script work if you added the same hooks for the telgridfile and polyorder that are in place for, e.g., flatfile?  I.e., essentially make `sensfile` and these other command-line arguments mutually exclusive?  If so, I would expose those command-line arguments, and add the necessary if statements.

Reply:
Have removed these options.